src/index.ts

```diff
@@ -168,6 +169,21 @@ export class ERC725 {
 
     throw new Error(`Incorrect or unsupported provider ${providerOrRpcUrl}`);
   }
+
+  private getAddressAndProvider() {
```

Comment:
@Hugoo I just moved this function at the top, to group the `private` methods together.

Reply:
ok nice ty